### PR TITLE
[11.0][MIG] barcodes_generator_picking

### PR DESCRIPTION
**Pull request by xavierjimenez**
Migration of the module **barcodes_generator_picking** to version 11.0.

### CODE REVIEW

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Same here.

Ref : https://github.com/OCA/stock-logistics-barcode/pull/142#pullrequestreview-127911815

**Review by legalsylvain — APPROVED** [review]
thanks !

**Review by pedrobaeza — APPROVED** [review]

File: barcodes_generator_picking/README.rst [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+Images
+------
+
+* Icon of the module is based on the Oxygen Team work and is under LGPL licence:
+  http://www.iconarchive.com/show/oxygen-icons-by-oxygen-icons.org.html
+
+Contributors
+------------
+
+* Dave Lasley <dave@laslabs.com>
```
Comment (pedrobaeza) [inline barcodes_generator_picking/README.rst:84]: You can add yourself here.

Comment (legalsylvain) [reply barcodes_generator_picking/README.rst:84]: https://github.com/OCA/stock-logistics-barcode/pull/143/commits/9a70d915aa016e7682a0674211e81a84d8f45cf6 ?

**Pull request merged**